homeassistant/components/uvc/camera.py

```diff
@@ -109,10 +110,29 @@ def supported_features(self):
 
         return 0
 
+    @property
+    def state_attributes(self):
+        """Return the camera state attributes."""
+        attr = super().state_attributes
+        attr["connection_state"] = self._caminfo["state"]
```

Comment:
Any reason why this is not a binary sensor with the connection class?

Reply:
Because I hadn't thought of that :smile: Pulled from the PR and will open a new one with it as a binary sensor.